charts/stable/immich/templates/_secret.tpl

```diff
@@ -21,5 +21,10 @@ data:
   REDIS_PASSWORD: {{ .Values.redis.redisPassword | trimAll "\"" | b64enc }}
   {{- with .Values.immich.mapbox_key }}
   MAPBOX_KEY: {{ . | b64enc}}
+  {{- with .Values.immich.typesense_api_key }}
+  TYPESENSE_API_KEY: {{ . }}
+  {{- else }}
+  TYPESENSE_API_KEY: {{ randAlphaNum 32 | b64enc }}
+  {{- end }}
```

Comment:
Take care of this end, because this one should be on line 24 closing the line 22, e.g.:
```diff
...
  {{- with .Values.immich.mapbox_key }}
  MAPBOX_KEY: {{ . | b64enc}}
+ {{- end }}
  {{- with .Values.immich.typesense_api_key }}
  TYPESENSE_API_KEY: {{ . }}
  {{- else }}
  TYPESENSE_API_KEY: {{ randAlphaNum 32 | b64enc }}
  {{- end }}
- {{- end }}
...